README.md

```diff
@@ -46,7 +46,8 @@ $ build/Darwin.x86_64.Debug/game/game
 #### Ubuntu
 
 ```
-$ apt-get install cmake libsdl2-dev libsdl2-ttf-dev liblua5.2-dev libboost-system-dev libboost-filesystem-dev
+$ apt-get install cmake libsdl2-dev libsdl2-ttf-dev liblua5.2-dev
+$ apt-get install libboost-system-dev libboost-filesystem-dev libboost-test-dev
```

Comment:
Thank you for your changes. However could you update this to just run this script? 

```
ci/install_deps_ubuntu.sh
```

% I prefer to install all boost packages rather than all libs. Since we are likely to add more boost libraries often.

Reply:
OK :)